src/Raven.Server/Documents/Handlers/Processors/OngoingTasks/AbstractOngoingTasksHandlerProcessorForBackupDatabaseNow.cs

```diff
@@ -24,8 +24,9 @@ public override async ValueTask ExecuteAsync()
             var isFullBackup = RequestHandler.GetBoolValueQueryString("isFullBackup", required: false) ?? true;
             var operationId = RequestHandler.GetLongQueryString("operationId", required: false) ?? GetNextOperationId();
             var startTime = RequestHandler.GetDateTimeQueryString("startTime", required: false);
+            var inProgressOnAnotherShard = RequestHandler.GetBoolValueQueryString("inProgressOnAnotherShard", required: false) ?? false;
```

Comment:
I do have a feeling that it would be better to have a dedicated endpoint to check if backup is running and what is the operationId of that backup, if not, the code gets a but messy IMO

Reply:
Just a heads up that if we add a dedicated endpoint then this specific parameter `inProgressOnAnotherShard` will still be needed in order for the backup to know not to run on certain shards. What it *will* save is the usage of `InvalidOperationId`, so the code will be more understandable